internal/database/models/models.go

```diff
@@ -1,25 +1,31 @@
 package models
 
 import (
+	"time"
+
 	"gorm.io/gorm"
 )
 
 type Author struct {
```

Comment:
removed `id` field from `Author`:

if we have `id` in Author, we have to set `auto_increment=True` (so that we don't have to manually pass in the `id` when inserting a new author). but for some reason, any integer column with `auto_increment` automatically becomes a primary key (and we don't want that). 

soo I just got rid of `id` column

i don't fully understand the magic that is happening here, does anyone know better?